### PR TITLE
Feature/1568 test zip

### LINKED ISSUE #1568: use more realistic zip codes in ADDR_Validator_TEST
in testContactZipOnly it inserts a text value into MailingPostalCode.  This ran afoul of a client routine that was expecting a well-formatted zipcode.  Their code should be more tolerant of non-well-formed data, but in the meantime, unless that text value is needed, could your code use more realistic zip data?  Seems like it would be less likely to run afoul of client code, validation rules, etc if it did.

### LINKED ISSUE #1626: Household Conversion Script Disregards Account Address Setting
When performing a Household Account migration using the Conversion Script (npsp__CONV_Account_Conversion) - currently, the Address settings are not taken into account and an Address record is created always. The conversion process should take the Address settings into account.

### PR DESCRIPTION
**Pull request by njjc**
# Warning
# Info
# Issues
Fixes #1626 
Fixes #1568

### CODE REVIEW

**Comment by njjc** [comment]
@davidhabib ready for review, and merge once build passes. I recommend adding ?w=1 to the end of the file review URL to ignore a bunch of whitespace changes.

**Comment by davidhabib** [comment]
CONV_Account_Conversion_BATCH_TEST.cls line 42 needs to have strTestOnly set back to '*' and the build hasn't run by the way.  you might want to manually kick it off after you fix this.

**Comment by davidhabib** [comment]
CONV_Account_Conversion_BATCH_TEST.cls line 294. do you still need to upsert the settings given our new findings about the settings facade and batches during tests?

File: src/classes/CONV_Account_Conversion_BATCH_TEST.cls [hunk 1]
```diff
@@ -39,13 +39,13 @@ public with sharing class CONV_Account_Conversion_BATCH_TEST {
     /*******************************************************************************************************
     * @description Sets the test methods in the class you want to run for development testing purposes
     */
-    private static string strTestOnly = '*';
+    private static string strTestOnly = 'testContactDisableAddresses';
```
Comment (davidhabib) [inline src/classes/CONV_Account_Conversion_BATCH_TEST.cls]: this needs to be put back to '*'

File: src/classes/CONV_Account_Conversion_BATCH_TEST.cls [hunk 2]
```diff
@@ -283,10 +283,87 @@ public with sharing class CONV_Account_Conversion_BATCH_TEST {
...
     /*******************************************************************************************************
+    * @description Tests the conversion process for Contacts with two addresses, but address verification disabled
+    */
+    public testMethod static void testContactDisableAddresses(){
+        if (strTestOnly != '*' && strTestOnly != 'testContactDisableAddresses') return;
+
+        //setup individual model and test account creation
+        npe01__Contacts_And_Orgs_Settings__c contactsSettings = UTIL_CustomSettingsFacade.getContactsSettingsForTests(new npe01__Contacts_And_Orgs_Settings__c(Household_Account_Addresses_Disabled__c=true, npe01__Account_Processor__c = CAO_Constants.ONE_TO_ONE_PROCESSOR));
+        //inserting settings so they are available for the new batch context
+        upsert contactsSettings;
```
Comment (davidhabib) [inline src/classes/CONV_Account_Conversion_BATCH_TEST.cls]: no longer needed given our findings on batch and contexts?

**Pull request merged**